src/map/modifier.h

```diff
@@ -553,6 +552,9 @@ enum MODIFIER
     MOD_HARVESTING_RESULT         = 0x201, // Improves harvesting results (modId = 513)
     MOD_LOGGING_RESULT            = 0x202, // Improves logging results (modId = 514)
     MOD_MINNING_RESULT            = 0x203, // Improves mining results (modId = 515)
+	MOD_ABSORB_DMG_TO_MP		  = 0x204, // (modId = 516)  Unlike PLD gear mod, works on all damage types (Ethereal Earring)

```

Comment:
I moved it to be consistent in the file. I made a new modifier just to find out 204 was taken somewhere else. I'll move it back to where it was at.